src/backend/marsha/core/tasks/timed_text_track.py

```diff
@@ -62,7 +62,7 @@ def convert_timed_text_track(timed_text_track_pk, stamp):
         prefix_destination = timed_text_track.get_videos_storage_prefix(stamp)
 
         with video_storage.open(source, "rt") as timed_text_file:
-            timed_text = timed_text_file.read()
+            timed_text = timed_text_file.read().replace("\ufeff", "")
             reader = detect_format(timed_text)
```

Comment:
Looks good as a workaround, but I feel this should be a contribution to the library which provide `detect_format` nope ?

Reply:
I don't know... pycaption accept a string in input, now how you retrieve it. I will open an issue to know if there are interested for this contribution

Reply:
Issue created: https://github.com/pbs/pycaption/issues/341